pkgs/applications/graphics/cloudcompare/default.nix

```
@@ -0,0 +1,86 @@
+{
+  stdenv,
+  fetchFromGitHub,
+
+  cmake,
+
+  dxflib,
```

usually see
```suggestion
, fetchFromGitHub

, cmake,

, dxflib
```

also the newlines look kind of odd...

---

Personally I believe that trailing `,` (as Python does) is better, because it is more diff-friendly (creating less merge conflicts) and that nixpkgs should move to that approach overall.

But maybe it's better to do that in a mass refactor, so I'll switch it here.

I used the newlines so that I can easily alphabetically sort deps but don't mix in higher-level things like `fetch*` and `cmake` into it.